packages/ui/src/components/Button/extendedButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button as DefaultButton } from 'antd'
+import classNames from 'classnames'
+import React from 'react'
+
+import { MoreHorizontal } from '@open-condo/icons'
+
+import { ButtonProps } from './button'
+
+
+const BUTTON_CLASS_PREFIX = 'condo-btn'
+
+export type ExtendedButtonProps = Omit<ButtonProps, 'icon' | 'href'>
```

Comment:
style and others still here :)

 (That's why I've suggested to use Condo Button with children type error suppression, so you can share common props, logic, types and modificators...)

Reply:
Oh, I've forgot, that we're not exporting it. So it's fine now...

(But placing it with styles closer to DropdownButton itself can help imho)

Reply:
`ButtonProps` from `CondoButton`)

Reply:
I'll add a note that this button is for internal use.

Reply:
moved to `Dropdown/button/...`